report/report.go

```diff
@@ -1,24 +1,61 @@
 package report
 
 import (
+	"encoding/json"
 	"html/template"
 	"log"
 	"os"
+	"path/filepath"
+	"strings"
 	"talisman/detector"
 )
 
+const htmlFileName string = "talisman_reports/report.html"
+const jsonFileName string = "talisman_reports/report.json"
 // GenerateReport generates a talisman scan report in html format
-func GenerateReport(r *detector.DetectionResults) {
+func GenerateReport(r *detector.DetectionResults, directory string) string {
+
+	var path string
+	var htmlFilePath strings.Builder
+	var jsonFilePath strings.Builder
+	if directory == "" {
```

Comment:
Why not specify "talisman_reports" in the default param value instead of checking for empty string here ?

Reply:
Have made the necessary modifications.